### PR TITLE
feat: rename /admin/form endpoint to /admin/forms, correct dir structure of /api/v3 routes

### PR DESCRIPTION
**Pull request by karrui**
## Problem

API endpoint for recently merged #1355 should be pluralized to follow REST resource conventions when the resource is in a collection. Thus, the plural form of `form` should be used for the new `/api/v3/admin/forms` endpoint introduced.

This PR also cleans up the directory structure of /api/v3 routes, giving the correct nested express routers

> This should be merged before the next release to ensure backwards compatibility. If it is not merged before that... I guess we will need to live with the old-new endpoint....

### CODE REVIEW

**Comment by liangyuanruo** [comment]
Btw can I also suggest separating the `ApiRouter` into two, for `/api` and `/v3`?

**Review by liangyuanruo — APPROVED** [review]

**Comment by karrui** [comment]
> Btw can I also suggest separating the `ApiRouter` into two, for `/api` and `/v3`?

Noted. Will do it in this PR

File: src/app/routes/admin/form/__tests__/admin-form.routes.spec.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ import { AdminFormRouter } from '../admin-form.routes'
 
 const UserModel = getUserModel(mongoose)
 
-const app = setupApp('/admin/form', AdminFormRouter, {
+const app = setupApp('/admin/forms', AdminFormRouter, {
```
Comment (liangyuanruo) [inline src/app/routes/admin/form/__tests__/admin-form.routes.spec.ts:18]: change the folder name as well?

**Comment by karrui** [comment]
Updated directory structure and split `ApiRouter` to `ApiRouter` and `V3Router`

**Review by liangyuanruo — APPROVED** [review]

**Pull request merged**